### PR TITLE
Javascript section

### PR DESCRIPTION
**Pull request by CrackedYoda**
Use basic JavaScript syntax.
    Use JavaScript to manipulate DOM elements.
    Use JavaScript events.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -17,14 +17,65 @@ body {
...
+    filter: blur(5px);
+  }
+
+  .logo.inactive {
+    filter: blur(9px);
+  }
+
+  .nav-menu.cancel {
+    content: url(/cancel.svg);
+  }
```
Comment (AlphaNtihinduka) [inline style.css:73]: - [ ] Please ensure that you stick to the Figma template design as much as possible. For this case, I suggest that you ensure that the cancel icon is displayed as seen in the Figma template design 😎 
Your design 😎 
image

Figma design 😎 
image

File: index.html [hunk 2]
```diff
@@ -15,27 +15,27 @@
...
           <a href="http://" class="alogo"> <h1 class="logo">My Logo</h1> </a>
           <ul class="nav-list">
-            <li class="nav-list-items">Portfolio</li>
-            <li class="nav-list-items">About</li>
-            <li class="nav-list-items">Contact</li>
+            <li class="nav-list-items"> <a href="#work" class="nav-list-items" id="portfoliolink"> Portfolio </a></li>
+            <li class="nav-list-items"><a href="#about" class="nav-list-items" id="aboutlink">About</a></li>
+            <li class="nav-list-items"><a href="#form" class="nav-list-items" id="contactlink">Contact</a></li>
           </ul>
           <img src="hamburger.svg" alt="hamburger-menu" class="nav-menu" />
```
Comment (AlphaNtihinduka) [inline index.html:26]: - [ ] I have noticed that after clicking on the humberger menu, and clicking on the cancel after, the humberger icon can no longer be displayed. Kindly ensure to display the icon at all the scenarios to have a professional and userfriendly portifolio.

Your design 😎 
image

File: menu.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+const contact = document.getElementById('contactlink');
+
+hamburger.addEventListener('click', () => {
+  navBar.classList.toggle('active');
+  document.querySelector('.nav-bar').classList.toggle('activecolor');
+  hamburger.classList.toggle('cancel');
+  document.querySelector('.headline').classList.toggle('blur');
+  document.querySelector('.work').classList.toggle('blur');
+  document.querySelector('.about').classList.toggle('blur');
+  document.querySelector('.form').classList.toggle('blur');
```
Comment (AlphaNtihinduka) [inline menu.js]: - [ ] Please ensure the blur effect is only applied to the mobile menu. This means blur effect even remains sticked when you scroll. To make it more clear, the portifolio sections should be seen through the blur effect with the mobile menu links still seen on the screen. Kindly apply the changes to match the Figma design and have portifolio with better UI ☺️.

Your design 😎 
image

Original design 😎 
image

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hello Team    🙋‍♂️
Good job working on mobile menu. You have done the design to do the expectation and incredibly well. 👏  

TO HIGHLIGHT:
- The mobile menu appears when user clicks (or taps) the hamburger button.
- On mobile the hamburger menu button is visible.
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Required changes :recycle: 
- Kindly consider the inline review comments ☺️.

## Optional suggestions
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Review by roshan-bajgain — APPROVED** [review]
Hi @CrackedYoda, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- GitHub flow is followed. ✔️
- Good job on implementing the changes ✔️ 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**